### ISSUE TITLE
Outdated version of the kubectl client

### ISSUE THREAD

**Issue by sampeka** [codeship-library/google-cloud-deployment#5]
Hey there

For some reason the `kubectl` client installed is outdated, which means we don't have `kubectl apply` which is the recommended way for updating deployments.

If you run `kubectl version` inside this container you'll get back:

```
Client Version: version.Info{Major:"1", Minor:"0", GitVersion:"v1.0.6", GitCommit:"388061f00f0d9e4d641f9ed4971c775e1654579d", GitTreeState:"clean"}
Server Version: version.Info{Major:"1", Minor:"3", GitVersion:"v1.3.7", GitCommit:"a2cba278cba1f6881bb0a7704d9cac6fca6ed435", GitTreeState:"clean"}
```

The latest version of the client is`v1.3.7`.

Its strange, because I can see you have a run directive for `gcloud components update kubectl` inside the Dockerfile. Is the image stored on dockerhub the same as what's here on github?

**Comment by sampeka** [comment]
ping @mlocher

**Comment by selslack** [comment]
Any updates on this one?

**Comment by selslack** [comment]
@mlocher ping

How I can contribute to make this happen _really_ fast (I see you merge only pulls from contributors)? We use `gcloud components update kubectl` hack to make it work but after today's update it just didn't work:

```
ERROR: gcloud failed to load: No module named apitools.base.protorpclite
```

Can I expect help from you guys anytime soon?

**Comment by naartjie** [comment]
Hi there, It looks like I'm encountering this issue as well.

I'm trying to deploy to GKE using `codeship/google-cloud-deployment`, and the error I get suggests that `kubectl` is at quite an old version, as `set image` is not a valid command. 

Should we rather be using the newwe `codeship/gcr-dockercfg-generator` image for GKE?

**Comment by naartjie** [comment]
Yup, confirmed, it's quite an old `kubectl` version:

```shell
$ kubectl version -c
Client Version: version.Info{Major:"1", Minor:"0", GitVersion:"v1.0.6", GitCommit:"388061f00f0d9e4d641f9ed4971c775e1654579d", GitTreeState:"clean"}
```

**Comment by naartjie** [comment]
Confirmed, this is fixed with #6.

**Comment by mlocher** [comment]
We also added regular tasks to rebuild the image, so this shouldn't happen again :)
